doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,149 @@
+# ADR 1: Record architecture decisions
+
+Date: 2022-06-08
+
+## Status
+
+Draft
+
+## Context
+
+We are in search for a means to document our architectural and design decisions
+for all of our components.
```

Comment:
I wonder if the word "search" is really what you want to say here. How about something like "We have determined that it is valuable to document our architectural and design decisions..." Or "We have decided to use the practice known as "architectural design records (ADR)" ...

Reply:
This is the context, so it's a description of the issue. Formulations like `We have determined..` should be in the decision section, no?